src/OrbitGl/App.cpp

```diff
@@ -2269,6 +2270,46 @@ orbit_base::Future<ErrorMessageOr<void>> OrbitApp::UpdateProcessAndModuleList()
       });
 }
 
+void OrbitApp::LoadAllSymbols() {
+  std::vector<const ModuleData*> all_modules = module_manager_->GetAllModuleData();
+
+  auto ggpvlk_it = std::find_if(
+      all_modules.begin(), all_modules.end(),
+      [](const ModuleData* module) { return absl::StrContains(module->file_path(), "ggpvlk.so"); });
+  if (ggpvlk_it != all_modules.end()) {
+    const ModuleData* ggpvlk_module{*ggpvlk_it};
+    if (!ggpvlk_module->is_loaded()) {
+      std::ignore = RetrieveModuleAndLoadSymbols(ggpvlk_module);
+      all_modules.erase(ggpvlk_it);
+    }
+  }
+
+  const ProcessData* process = GetTargetProcess();
+  if (process == nullptr) {
+    // Orbit is not currently connected, so this uses the process from capture data, which then is
+    // from the capture that was loaded from file.
+    process = GetCaptureData().process();
+  }
+  ORBIT_CHECK(process != nullptr);
+  auto main_module_it =
+      std::find_if(all_modules.begin(), all_modules.end(), [process](const ModuleData* module) {
+        return absl::StrContains(module->file_path(), process->full_path());
+      });
+  if (main_module_it != all_modules.end()) {
+    const ModuleData* main_module{*main_module_it};
+    if (!main_module->is_loaded()) {
+      std::ignore = RetrieveModuleAndLoadSymbols(main_module);
+      all_modules.erase(main_module_it);
+    }
+  }
```

Comment:
This is similar to the ggpvlk code path. Could this be shared?

Reply:
done